simple_calculator/myscript.js

```diff
@@ -2,10 +2,16 @@ const showNumber = document.getElementById("result")
 let resultNumber = Number(showNumber.innerText)
 
 document.addEventListener("click", function(e) {
-    console.log("clicked")
+    
     if (e.target.className.includes("cal-button")) {
+        console.log("cal-button clicked")
         resultNumber = resultNumber + Number(e.target.innerText)
         showNumber.innerText = resultNumber
         console.log(showNumber.innerText)
     }
+
+    if (e.target.className.includes("reset")) {
```

Comment:
(reset이라는 id를 div에 박았다고 가정하고) 아래와 같이 수정하면 동작 안되나요?

```suggestion
    if (e.target.id === "reset") {
```
(위와 같이 수정했을때 잘 된다면,`e.target.id.includes("reset")`해도 잘될듯.. 물론 굳이 그럴필욘 없지만)

Reply:
@HyunSangHan 
오 되네요!!